Sources/Plasma/PubUtilLib/plNetClient/plNetLinkingMgr.cpp

```diff
@@ -84,18 +84,24 @@ struct NlmOp {
     NlmOp (const ENlmOp & op)
     : opcode(op)
     { }
+
+    virtual ~NlmOp() = default;
 };
 
 struct NlmNoOpOp : NlmOp {
     NlmNoOpOp ()
     : NlmOp(kNlmOpNoOp)
     { }
+
+    virtual ~NlmNoOpOp() = default;
```

Comment:
This doesn't need to be declared -- the base is already virtual, and this override is a no-op.